v1/summary.yaml

```diff
@@ -27,7 +27,7 @@ paths:
         sentences, as well as information about a thumbnail that represents
         the page.
 
-        Stability: [experimental](https://www.mediawiki.org/wiki/API_versioning#Experimental)
+        Stability: [stable](https://www.mediawiki.org/wiki/API_versioning#Stable)
```

Comment:
The 'stable' doesn't mean the endpoint result is fixed forever, we're still gonna tweak it, just in a backwards-compatible way. Summary is used in so many places now that we have absolutely no possibility to make backwards incompatible changes or remove it..